tasks/broad/RNAWithUMIsTasks.wdl

```diff
@@ -98,11 +98,14 @@ task Fastp {
     File adapter_fasta = "gs://gcp-public-data--broad-references/RNA/resources/Illumina_adapters.fasta"
 
     String docker = "us.gcr.io/broad-gotc-prod/fastp:1.0.0-0.20.1-1649253500"
-    Int memory_mb =  "16384"
+    Int memory_mb =  ceil(1.5*size(fastq1, "MiB")) + 8192 # Experimentally determined formula for memory allocation
     Int disk_size_gb = 5*ceil(size(fastq1, "GiB")) + 128
+    File monitoring_script = "gs://broad-dsde-methods-monitoring/cromwell_monitoring_script.sh"
```

Comment:
I know we need to add the monitoring script this way in order to use it on Terra, but is this script at least publicly accessible? If not can we use a public copy? (same comment applies to other instances as well)

Reply:
I don't think it's publicly accessible as is; is there a place where we can put it?

Reply:
Good question :) I'm not sure. I'll have to figure that out. This should be OK in the meantime